content/topics/file-and-directory-naming/general/_index.md

```diff
@@ -0,0 +1,19 @@
+--- 
+content_type: topic
```

Comment:
Please add these to the relevant syllabuses found in the folder: content/syllabuses/